### DISCUSSION TITLE
How to style parent element without leaking style to children

### DISCUSSION THREAD

**Discussion by leonardopn** [chakra-ui/chakra-ui#6668] [Q&A]
I'm making a card element that I'm going to add a border and some more rules, inside it I have some inputs, but when I apply the css properties to the parent element, some end up going to the inputs, but I just wanted to style the parent.
Is there any way I can style just this box?

![image](https://user-images.githubusercontent.com/39427966/189463960-2b6786d3-7885-4748-8db3-f39165dbc1ea.png)
![image](https://user-images.githubusercontent.com/39427966/189464072-26634899-6e2f-4c28-ad3e-2587665ae08e.png)

**Comment by divyanshyadav** [comment]
Happening for me as well.
